org.catacombing.kumo.metainfo.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2025 Christian Duerr <contact@christianduerr.com> -->
+<component type="desktop-application">
+    <id>org.catacombing.kumo</id>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>GPL-3.0-only</project_license>
+    <name>Kumo</name>
+    <summary>A Wayland Mobile Browser</summary>
+    <developer id="org.catacombing">
+        <name>Christian Duerr</name>
+    </developer>
+    <description>
+        <p>
+            Kumo is a web browser with a UI focused on portrait-mode touchscreen
+            mobile devices. It is optimized to run on low-end hardware with a
+            limited battery life.
+        </p>
+    </description>
+    <launchable type="desktop-id">org.catacombing.kumo.desktop</launchable>
+    <provides>
+        <id>Kumo.desktop</id>
+    </provides>
+    <branding>
+        <color type="primary" scheme_preference="dark">#752a2a</color>
+        <color type="primary" scheme_preference="light">#752a2a</color>
+    </branding>
+    <content_rating type="oars-1.1"/>
+    <url type="homepage">https://github.com/catacombing/kumo</url>
+    <url type="bugtracker">https://github.com/catacombing/kumo/issues</url>
+    <url type="vcs-browser">https://github.com/catacombing/kumo</url>
+    <screenshots>
+        <screenshot type="default">
+            <image>https://raw.githubusercontent.com/catacombing/kumo/refs/heads/master/screenshots/main_ui.png</image>
+            <caption>Kumo with the pine64.org landing page loaded</caption>
+        </screenshot>
+        <screenshot>
+            <image>https://raw.githubusercontent.com/catacombing/kumo/refs/heads/master/screenshots/tab_groups.png</image>
+            <caption>Tabs organized in a tab group</caption>
+        </screenshot>
+    </screenshots>
+    <releases>
+        <release version="0.0.0" date="2025-04-11">
+            <description>
+                <p>Preview release of Kumo</p>
+            </description>
+        </release>
+    </releases>
+    <recommends>
+        <control>touch</control>
+        <control>keyboard</control>
+        <control>pointing</control>
+    </recommends>
```

Comment:
I'm not sure if the order of these matters. While keyboard and pointing are technically supported and should work just fine, touch is definitely the primary target audience.

From what I've seen essentially anything that isn't a hard requirement should just go into "recommends" and there isn't really such a thing as tiered platforms (just support or no support).